src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -19,77 +20,52 @@
-  private Solenoid rotationSolenoid = new Solenoid(Constants.SHOOTER_ROTATION_SOLENOID_ID);
+  private VictorSPX shooterFollower = new VictorSPX(Constants.SHOOTER_SHOOTER_FOLLOWER_ID);
 
-  private double motorSpeed = 0.0;
+  private int setPoint = 125000;
 
   public Shooter() {
     shooterMaster.configFactoryDefault();
     shooterFollower.configFactoryDefault();
 
+    shooterMaster.setNeutralMode(NeutralMode.Coast);
+    shooterFollower.setNeutralMode(NeutralMode.Coast);
+
     shooterMaster.setInverted(false);
     shooterMaster.setSensorPhase(true);
 
     shooterFollower.setInverted(true);
     shooterFollower.follow(shooterMaster);
   }
-  // prints motor speed 
-  public void printMotorVelocity(){
-    System.out.println(shooterMaster.getSelectedSensorVelocity());
-  }
-  // sets the solenoid 
-  public void setAdjustmentSolenoid(Solenoid adjustmentSolenoid) {
-    this.adjustmentSolenoid = adjustmentSolenoid;
-  }
-  // sets the rotational solenoid 
-  public void setRotationSolenoid(Solenoid rotationSolenoid) {
-    this.rotationSolenoid = rotationSolenoid;
-  }
-
-  //increaseMotorSpeed is only to be used for testing
-  public void increaseMotorSpeed(){
-    motorSpeed += 0.01;
-    motorSpeed = motorSpeed >= 1.00 ? 1 : motorSpeed;
-    System.out.println("Motor ++ | " + motorSpeed);
-    setCurrentMotorSpeed();
-  }
-
-  //decreaseMotorSpeed is only to be used for testing
-  public void decreaseMotorSpeed(){
-    motorSpeed -= 0.01;
-    motorSpeed = motorSpeed <= -1.00 ? -1 : motorSpeed;
-    System.out.println("Motor -- | " + motorSpeed);
-    setCurrentMotorSpeed();
-  }
 
-  public void setCurrentMotorSpeed(){
-    shooterMaster.set(ControlMode.PercentOutput, motorSpeed);
+  public void testMotors(){
+    shooterMaster.set(ControlMode.PercentOutput, .5);
+    System.out.println("Voltage| "+shooterMaster.getMotorOutputVoltage());
+    System.out.println("Velocity| "+shooterMaster.getSelectedSensorVelocity());
   }
 
-  public void maintainRPM() {
-    if(shooterMaster.getSelectedSensorVelocity() < 125000){
+  public boolean maintainRPM() {
+    if(shooterMaster.getSelectedSensorVelocity() < setPoint){
       shooterMaster.set(ControlMode.PercentOutput, 1);
     } else {
       shooterMaster.set(ControlMode.PercentOutput, .85);
     }
+    System.out.println("Velocity| "+ shooterMaster.getSelectedSensorVelocity());
+    return shooterMaster.getSelectedSensorVelocity() > setPoint-5000;
   }
 
-  public void stopMotors(){
-    shooterMaster.set(ControlMode.PercentOutput, 0);
+  public void manualUp(){
+    //this code is unregulated and dumb...but who knows whether its even going to be used.
+    setPoint += 1000;
   }
 
-  public boolean getAtSpeed(){
-    return (shooterMaster.getSelectedSensorPosition() < 125000);
+  public void manualDown(){
+    //this code is unregulated and dumb...but who knows whether its even going to be used.
+    setPoint -= 1000;
```

Comment:
Error trap this so it doesn't go below 0% motor speed (have negative speed when we don't intend it to)